plugins/CoreHome/tests/Integration/Column/VisitLastActionTimeTest.php

```diff
@@ -58,6 +58,34 @@ private function makeRequest($request)
         return new Request($request);
     }
 
+    public function test_convertHourToHourInSiteTimezone_UTC()
+    {
+        $idSite = Fixture::createWebsite('2020-01-02 03:04:05');
+        $hourConverted = VisitLastActionTime::convertHourToHourInSiteTimezone(5, $idSite);
+        $this->assertEquals(5, $hourConverted);
+    }
+
+    public function test_convertHourToHourInSiteTimezone_WithTimezone()
+    {
+        $idSite = Fixture::createWebsite('2020-01-02 03:04:05', $ecommerce = 1, 'Site', $siteUrl = false,
+            $siteSearch = 1, $searchKeywordParameters = null,
+            $searchCategoryParameters = null, $timezone = 'Pacific/Auckland');
+        $hourConverted = VisitLastActionTime::convertHourToHourInSiteTimezone(5, $idSite);
+        $this->assertEquals(18, $hourConverted);
```

Comment:
Just out of curiosity: Will this test fail when the time is changed due to daylight savings? Guess Auckland has summer/winter time as well, right?

Reply:
Yes it will. I reckon it'll be easy to change though then. Do you maybe know a timezone that doesn't have daylight savings? happy to change it

Reply:
Asia shouldn't have daylight saving afaik. Guess you could use eg `Asia/Jakarta` (UTC+7)